pages/onboarding/index.tsx

```diff
@@ -59,16 +68,23 @@ const OnboardingPage: NextPage<Props> = ({ user }: Props) => {
       router.push('/directory');
     } catch (error) {
       setIsLoading(false);
-      alert(error);
+      setOpenAlertError(error);
```

Comment:
> Error: Objects are not valid as a React child (found: object with keys {message, name, code, config, request, response}). If you meant to render a collection of children, use an array instead.

`setOpenAlertError()` expects a `ReactChild`, not an object. A string would be a valid `ReactChild` so it should be `error.message`...

That said, we could probably just put a generic error in here... doing so would be less confusing to non-technical users.